.ci.yaml

```diff
@@ -8,9 +8,40 @@
 enabled_branches:
   - master
 
+platform_properties:
+  windows:
+    properties:
+      caches: >-
+        [
+          {"name": "vsbuild", "path": "vsbuild"},
+          {"name": "pub_cache", "path": ".pub-cache"}
+        ]
+      dependencies: >
+        [
+          {"dependency": "certs"}
+        ]
+      device_type: none
+      os: Windows
+
 targets:
   - name: Windows Plugins
-    builder: Windows Plugins
-    postsubmit: false
+    recipe: plugins/plugins
```

Comment:
I consolidated this with the postsubmit "Windows Plugins master channel" which was equivalent